lib/slather/coverage_info.rb

```diff
@@ -71,7 +71,11 @@ def rate_branches_tested
     end
 
     def source_file_pathname_relative_to_repo_root
-      source_file_pathname.realpath.relative_path_from(Pathname("./").realpath)
+      test_path=source_file_pathname
+      if ENV["COVERAGE_PATH_EQUIVALENCE"]
+        test_path=Pathname("." + "#{source_file_pathname}".delete_prefix(ENV["COVERAGE_PATH_EQUIVALENCE"].delete_suffix(",.")))
```

Comment:
This only supports `COVERAGE_PATH_EQUIVALENCE=XXX,.`, i.e. relocation from a given source root (XXX) to the current directory.